simulator-core/src/test/java/com/evbox/everon/ocpp/simulator/station/component/chargingstation/IdentityVariableAccessorTest.java

```diff
@@ -0,0 +1,94 @@
+package com.evbox.everon.ocpp.simulator.station.component.chargingstation;
+
+import com.evbox.everon.ocpp.common.CiString;
+import com.evbox.everon.ocpp.simulator.configuration.SimulatorConfiguration;
+import com.evbox.everon.ocpp.simulator.station.Station;
+import com.evbox.everon.ocpp.v20.message.centralserver.*;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.stream.Stream;
+
+import static com.evbox.everon.ocpp.simulator.assertion.CiStringAssert.assertCiString;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.params.provider.Arguments.arguments;
+import static org.mockito.BDDMockito.given;
+
+@ExtendWith(MockitoExtension.class)
+class IdentityVariableAccessorTest {
+
+    private static final String STATION_IDENTITY_CODE = "EVB-12345678";
```

Comment:
Can you please move this field to StationConstants class? The purpose of this class is to share common constants, such as Station_id

Reply:
There is one already